apps/ex_nvr_web/lib/ex_nvr_web/live/device_live.html.heex

```diff
@@ -45,6 +45,20 @@
         />
       </div>
 
+      <.input
+        :if={@device_type == "ip"}
+        field={@device_form[:vendor]}
+        id="device_vendor"
+        type={if Mix.env() == :test, do: "text", else: "select"}
+        options={[
+          {"", nil},
+          {"Hikvision", "hikvision"},
+          {"Milesight Technology Co.,Ltd", "milesight"},
+          {"AXIS", "axis"}
+        ]}
+        label="Vendor"
+        placeholder="Choose your device vendor"
```

Comment:
Use `prompt` here to add an option like this `placeholder` and delete the `{"", nil}` option above. And no need to map the values. If the model is `Hivision` store it as `Hikvision` not `hikvision`.